packages/core/content-type-builder/admin/src/components/AttributeOptions/CustomFieldsList/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useCustomFields } from '@strapi/helper-plugin';
+import { Box } from '@strapi/design-system/Box';
+import { Grid, GridItem } from '@strapi/design-system/Grid';
+import { KeyboardNavigable } from '@strapi/design-system/KeyboardNavigable';
+import { Stack } from '@strapi/design-system/Stack';
+import EmptyAttributes from '../EmptyAttributes';
+import CustomFieldOption from '../CustomFieldOption';
+import getPadding from '../utils/getPadding';
+
+const CustomFieldsList = () => {
+  const customFields = useCustomFields();
+  const registeredCustomFields = Object.entries(customFields.getAll());
+
+  if (!registeredCustomFields.length) return <EmptyAttributes />;
+
+  // Sort the array alphabetically by customField name
+  const sortedCustomFields = registeredCustomFields.sort((a, b) =>
+    a[1].name > b[1].name ? 1 : -1
+  );
+
+  return (
+    <KeyboardNavigable tagName="button">
+      <Stack spacing={8}>
+        <Grid gap={0}>
+          {sortedCustomFields.map(([uid, customField], index) => {
+            const { paddingLeft, paddingRight } = getPadding(index);
+
+            return (
+              <GridItem key={uid} col={6} style={{ height: '100%' }}>
+                <Box
+                  paddingLeft={paddingLeft}
+                  paddingRight={paddingRight}
+                  paddingBottom={1}
+                  style={{ height: '100%' }}
+                >
+                  <CustomFieldOption key={uid} uid={uid} customField={customField} />
```

Comment:
I don't think using `uid` as a prop here is a good idea, I think it makes more sense to keep this name for the contentType.uid or the component.uid it will make it easier to maintain in the future.

WDYT about `customFieldUid` instead?

Reply:
Sounds good to me, what I find unfortunate is that `AttributeIcon` inside `<CustomFieldOption>` will have a `customField` prop that takes `customFieldUid` I think it would have been nice to keep that prop naming convention consistent all the way down...

Reply:
Not too late to update your code to use `customFieldUid` instead but I would suggest you to keep `uid` only for the component.uid and the `contentType.uid` as the `uid` of the `forTarget`